runtime-data/wsl-default-settings/autobuild/build

```diff
@@ -0,0 +1,12 @@
+mkdir -pv "$PKGDIR"/etc
+
+abinfo "Writing default settings to wsl.conf ..."
+cat > "$PKGDIR"/etc/wsl.conf << EOF
+[boot]
+# Enable Systemd support
+systemd=true
+
+[automount]
+# Enable rootfs sharing between WSL distros support
+crossDistro = true
```

Comment:
This option seems to be [abandoned](https://github.com/microsoft/WSL/issues/4577#issuecomment-685173446).